### PR TITLE
(IAC-451) Create initial sample input file for SingleStore Azure cluster

### PR DESCRIPTION
**Pull request by dhoucgitter**
Initial draft of best practice terraform inputs for the creation of SingleStore compatible Azure cluster for a Viya4 deployment.

### CODE REVIEW

File: examples/sample-input-singlestore.tfvars [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    "machine_type" = "Standard_E8ds_v5"         
+    "os_disk_size" = 200
+    "min_nodes" = 0
+    "max_nodes" = 7
+    "max_pods" = 110
+    "node_taints" = ["workload.sas.com/class=singlestore:NoSchedule"]
+    "node_labels" = {
+      "workload.sas.com/class" = "singlestore"
+    }
+  },
```
Comment (thpang) [inline examples/sample-input-singlestore.tfvars:54]: I would suggest adding this at the end given the issues we've seen with AWS when/if we need to support ss there. I'd also remove the extra space.

**Review by thpang — CHANGES_REQUESTED** [review]
Comments for change.

File: examples/sample-input-singlestore.tfvars (hunk 1 above)
Comment (dhoucgitter) [reply examples/sample-input-singlestore.tfvars:54]: point taken, thanks for the suggestion

**Review by thpang — APPROVED** [review]

**Comment by dhoucgitter** [comment]
@thpang deleting this branch per request and replacing it with a branch named singlestore which I will seed with udpated sample-input-singlestore.tfvars file.

**Pull request closed without merging**